css/style.css

```
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    border: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #eee;
+}
+
+#header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1rem 2rem;
+    position: fixed;
+    width: auto;
+    background-color: #eee;
+}
+
+#header>img {
+    width: 30%;
+}
+
+#features >div {
+    margin: 2rem;
+}
+
+#nav-bar>ul {
+    list-style: none;
+    display: flex;
+    gap: 2rem;
+}
+
+.nav-link {
+    text-decoration: none;
+}
+
+footer {
+    background-color: #ddd;
+}
+
+#how-it-works {
+    width: 300px;
+    margin: auto;
+}
```

```suggestion
#how-it-works {
  width: 300px;
  margin: auto; /* there's 2 indentations here instead of 4 */
}
```

I think adding linters will help your code be a little cleaner and easier to read. Try checking out this [helpful documentation](https://github.com/microverseinc/linters-config) about them.